common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2234,7 +2234,9 @@ public static enum ConfVars {
         "If this is set to true the URI for auth will have the default location masked with DEFAULT_TABLE_LOCATION"),
     HIVE_ICEBERG_ALLOW_DATAFILES_IN_TABLE_LOCATION_ONLY("hive.iceberg.allow.datafiles.in.table.location.only", false,
         "If this is set to true, then all the data files being read should be withing the table location"),
-
+    HIVE_ICEBERG_COMPACTOR_POOL_NAME("hive.iceberg.compactor.pool.name", "default",
```

Comment:
let's not create Hive config for that. can we use tbl properties instead?

Reply:
We can set `hive.compactor.worker.pool` table property in HiveIcebergMetaHook during table creation for Iceberg tables, but to what value if it is not defined in Conf? To something hardcoded?

Reply:
Fixed. Now if the pool name is not provided in the compaction command it attempts to get it from the table property `hive.compactor.worker.pool`, otherwise compacts on the default pool.